.github/workflows/scala.yml

```diff
@@ -2,14 +2,14 @@ name: Scala CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, 2.x ]
```

Comment:
Maybe cleaner to let this be a pure merge PR and remove those afterwards in a separate PR.